File: samples/setup/Setup.UWP/ViewModels/HomeViewModel.cs
using System;
using Caliburn.Micro;

namespace Setup.UWP.ViewModels
{
    public class HomeViewModel : Screen
    {
        private string _title;

        public string Title
        {
            get => _title;
            set
            {
                _title = value;
                NotifyOfPropertyChange("Title");
            } 
        }

        public HomeViewModel()
        {
            Title = "Welcome to Caliburn Micro in UWP";
        }
    }
}

File: samples/setup/Setup.UWP.VB/ViewModels/HomeViewModel.vb

Imports Caliburn.Micro

Public Class HomeViewModel
    Inherits Screen

    Private _title1 As String

    Public Property Title As String
        Get
            Return _title1
        End Get
        Set
            _title1 = Value
        End Set
    End Property

    Public Sub New()
        Title = "Welcome to Caliburn Micro UWP using VB"
    End Sub
End Class

